README.md

```diff
@@ -154,6 +154,30 @@ kubectl delete namespace kogito-workflows
 make undeploy
 ```
 
+## Use Workflow Development Mode
+In the development mode, a user can edit and reload the files using the quarkus dev mode.
+To enable the dev mode add this annotation in the KogitoServerlessWorkflow CR:
```

Comment:
## Use Workflow Development Mode
In the development mode, a user can edit and reload the files using the quarkus dev mode.

To enable the dev mode add this annotation in the KogitoServerlessWorkflow CR: